great_expectations/core/batch.py

```diff
@@ -286,7 +290,7 @@ def __init__(
         limit: Optional[int] = None,
         batch_spec_passthrough: Optional[dict] = None,
     ):
-        self._validate_batch_request(
+        self._validate_init_parameters(
```

Comment:
Same concern as in the comment above.  I have held the understanding that `BatchRequest` had to be strictly validated.  Happy to discuss.

Reply:
@Shinnnyshinshin This change is still there, it appears.  Do we really want to rename `_validate_batch_request()` into `_validate_init_parameters()`?..  I am not seeing the critical need to do so...  Happy to discuss.

Reply:
settled